### ISSUE TITLE
would 'Disable cudnn for batch_norm' influence other code?  

### ISSUE THREAD

**Issue by lilichu** [Microsoft/human-pose-estimation.pytorch#93]
hi! 
I disable cudnn for batch_norm. If I use other code such as mask rcnn, will it influences the result of mask rcnn?
thanks!

**Comment by leoxiaobin** [comment]
For my understanding, it will not. Disable cudnn for batch_norm only means that it will not use the cudnn's implement of BN.
